sixtyfps_compiler/layout.rs

```diff
@@ -454,3 +454,34 @@ pub fn layout_info_type() -> Type {
         node: None,
     }
 }
+
+/// Get the implicit layout info of a particular element
+pub fn implicit_layout_info_call(elem: &ElementRc, orientation: Orientation) -> Expression {
+    if matches!(&elem.borrow().base_type ,
+                    Type::Builtin(base_type) if base_type.name == "Rectangle")
```

Comment:
We have this pattern in a few places, might become worth a helper function in Element